recipes/pyflowy/recipe.yaml

```diff
@@ -0,0 +1,88 @@
+schema_version: 1
+
+context:
+  name: pyflowy
+  version: 0.0.1
+  build_num: 0
```

Comment:
Instead of using `context` to set jinja variables for `build_num` and `shasum`, just set them explicitly below.